README.md

```diff
@@ -121,6 +121,7 @@ _Web Analytics_
 - [Matomo](https://matomo.org/) - Leading open-source analytics platform that gives you more than just powerful analytics, formerly known as Piwik. ([Source Code](https://github.com/matomo-org/)) `GPL-3.0` `PHP`
 - [Open Web Analytics](http://www.openwebanalytics.com/) - Google Analytics and Matomo alternative. ([Source Code](https://github.com/padams/Open-Web-Analytics)) `GPL-2.0` `PHP`
 - [Rakam](https://rakam.io/) - Custom analytics platform that allows you to create your own analytics services. Integrate with any data source (web, mobile, IoT etc.), analyze data with SQL and create dashboards. ([Source Code](https://github.com/rakam-io/rakam)) `Apache-2.0` `Java`
+- [Rudder](https://rudderlabs.com/) - Customer Data Infrastructure software for collecting, storing, processing and routing event data from apps and websites to dozens of marketing tools & analytics platforms (open-source alternative to Segment et.al.) ([Source Code](https://github.com/rudderlabs/rudder-server/)) `SSPL` `Go`
```

Comment:
- missing final dot after `et.al.)`
- SSPL-1.0 (https://spdx.org/licenses/SSPL-1.0.html) is not in the list of licenses yet, can you add it?

Reply:
Added SSPL
Added final dot after the bracket
Removed extra dot from et.al.